include/openrave/planner.h

```diff
@@ -62,7 +62,9 @@ enum PlannerStatusCode
 enum PlanningOptions
 {
     PO_NoStatusDetail = 1, ///< if set, then do not output any PlannerStatus details, just the finish code. This allows system to be faster.
-    PO_AddCollisionStatistics = 2 /// If set, Fill in collision to aid in planning failure debugging. This will slow down the system.
+    PO_AddBiRRTCollisionInfo = 2, /// If set, then will fill in collision information to aid in BiRRT planning failure debugging. This will slow down the system.
+    PO_AddLinearPlannerCollisionInfo = 4, /// If set, then will fill information to aid in linear planning failure debugging. This will slow down the system.
+    PO_AddWorkspacePlannerCollisionInfo = 0x8 /// If set, fill in collision to aid in IK filering failure debugging. This will slow down the system.
```

Comment:
Why `0x8` but not the others? Perhaps `(1 << 0)`, `(1 << 1)` is easier to understand.